### ISSUE TITLE
Color contrast in spec

### ISSUE THREAD

**Issue by michael-n-cooper** [w3c/wcag21#156]
Filed by private email.

I notice that the green text on green background (for the Editor's Notes) is not meeting color contrast. It is only 2.37 to 1.  foreground color: #22bb22, background color: #e9fbe9
Gentle Suggestion: I think if you change it to  foreground #008400/ background #e9fbe9

that it will pass.

P.S.  Also...the Respec-pill only has color contrast of 1.19 (foreground color: #ffe9bf, background color: #ffd280, font size: 8.3pt, font weight: normal)

<button id="respec-pill-warning" class="respec-info-button" aria-expanded="false" aria-haspopup="true" aria-controls="respec-pill-warning-modal" aria-label="Document warnings">1</button>

Gentle Suggestion: Would it be possible to change the white text in this respec pill to black?  

P.P.S.  Finally, the grey Respec text on the white button background only has a 4.4 to 1 ratio (foreground color: #787878, background color: #ffffff, font size: 8.3pt, font weight: normal) 

<button id="respec-pill" aria-controls="respec-menu" aria-expanded="false" aria-haspopup="true" aria-label="ReSpec Menu">ReSpec</button>

Gentle suggestion: If you bump up the grey text to #767676 it will pass.

**Comment by michael-n-cooper** [comment]
Note many of these issues are with Respec or W3C styles and should be filed there.

**Comment by awkawk** [comment]
We should make sure to do an accessibility review of the TR document.

**Comment by steverep** [comment]
@awkawk, suggest changing title if you want this issue to track an accessibility review.

See w3c/respec#425 and w3c/respec#827 for other issues I've raised that still are not fixed in our publications.

**Comment by awkawk** [comment]
@michael-n-cooper Do we have a publication checklist that we could just add "accessibility review of Spec" to and then close this issue?

**Comment by michael-n-cooper** [comment]
We could add accessibility review to the [QA checklist](https://www.w3.org/WAI/GL/wiki/WCAG_2.1_QA_Checklist) but I don't think it would advance accessibility. The problems we're encountering are with other tools we use, including the W3C TR styles that all TR documents must use, so a routine QA check of the guidelines won't help us.

We should use this issue as a trigger to get them fixed in other places. I've asked Roy to take a look at that. This may require looking at [Respec Accessibility Issues](https://github.com/w3c/respec/issues?q=is%3Aissue+is%3Aopen+label%3Aa11y) and the [TR styles](https://github.com/w3c/tr-design). We may or may not be able to fix some problems by override styles using our [guidelines CSS](https://github.com/w3c/wcag21/blob/master/guidelines/guidelines.css).

**Comment by jake-abma** [comment]
Looking at https://w3c.github.io/wcag21/guidelines/ there’s also a fail for the red vertical bar (W3C Editor’s Draft) at the left: https://www.w3.org/StyleSheets/TR/2016/logos/ED which has white text (#ffffff) on a red background with hex code: #ff1b00

Beside a fail for color: SC 1.4.3 we also have a failure of SC 1.4.5

Note: The image when scaling fastly looses sharpness as the image is very small, at https://www.w3.org/TR/UNDERSTANDING-WCAG20/ the image (when you still choose to use an image) is much bigger, staying sharper at zoom levels, see: https://www.w3.org/StyleSheets/TR/2016/logos/WG-Note

**Comment by ruoxiran** [comment]
For the green text on green background (for the Editor's Notes) is not meeting color contrast, I change the forground color in our guidelines CSS, change it to foreground #008400.

**Comment by michael-n-cooper** [comment]
I added Roy's  color suggestion to the guidelines.css. The red color is no longer being used. Further style issues, unless very clearly caused by WCAG-specific style choices, should be filed in the [TR Design repository](https://github.com/w3c/tr-design/issues).
